front/lib/api/assistant/conversation/destroy.ts

```diff
@@ -158,7 +157,9 @@ export async function destroyConversation(
   const conversationRes = await getConversationWithoutContent(
     auth,
     conversationId,
-    true
+    // We skip access checks as some conversations associated with deleted spaces may have become
+    // inaccessible, yet we want to be able to delete them here.
+    { includeDeleted: true, dangerouslySkipAccessCheck: true }
```

Comment:
`nit` if we can align with the same logic on agent configurations `dangerouslySkipPermissionFiltering`.

Reply:
You mean naming wise?

Reply:
yes

Reply:
(done)